rotkehlchen/chain/ethereum/modules/makerdao/decoder.py

```diff
@@ -525,5 +611,9 @@ def counterparties(self) -> list[CounterpartyDetails]:
                 identifier=CPT_MIGRATION,
                 label=MAKERDAO_LABEL,
                 image=MAKERDAO_ICON,
+            ), CounterpartyDetails(
+                identifier=CPT_SDAI,
+                label=MAKERDAO_LABEL,
```

Comment:
This should be a different label probably. Not makerdao label. Since this would read makerdao